spec/function.dd

```diff
@@ -2836,17 +2836,14 @@ $(H2 $(LNAME2 nogc-functions, No-GC Functions))
 
 $(H2 $(LNAME2 function-safety, Function Safety))
 
-        $(P $(I Safe functions) are functions that are statically checked
-        to exhibit no possibility of
-        $(DDSUBLINK glossary, undefined_behavior, $(I undefined behavior)).
```

Comment:
I think it's still useful to mention that safe functions cannot exhibit undefined behavior.

Reply:
> I think it's still useful to mention that safe functions cannot exhibit undefined behavior.

That's moved to the definition of safe interfaces. Safe functions have safe interfaces. Functions with safe interfaces cannot exhibit undefined behavior.

I don't think it's particularly useful to repeat that aspect of safe interfaces in the definition of safe (and trusted) functions. But I wouldn't mind if someone added it back.